.gitignore

```diff
@@ -208,6 +208,9 @@ rel/
 mambaforge/
 miniconda3/
 
+# in source build of conda pacakges
+conda/recipes/
```

Comment:
Won't this ignore all changes to the sources in `conda/recipes` too?

Reply:
Things in `.gitignore` are default ignored. You can still add them explicitly.